packages/esm-hiv-app/src/views/hts/home/summary-tiles/hts-summary-tiles.component.tsx

```diff
@@ -6,23 +6,20 @@ import {
   fetchPatientsFromObservationCodeConcept,
   fetchTodayClients,
 } from '@ohri/openmrs-esm-ohri-commons-lib';
-import {
-  finalHIVCodeConcept,
-  finalPositiveHIVValueConcept,
-  linkedToCareCodeConcept,
-  linkedToCareYesValueConcept,
-} from '../../../../constants';
+
 import styles from './summary-tile.scss';
 import { TodaysClientList } from './today-client-list-tile.component';
 import { PositiveInLast14Days } from './positive-in-last-14-days-list-tile.component';
 import { LinkedToCareInLast14Days } from './linked-to-care-in-last-14-days-list-tile.component';
 import { useTranslation } from 'react-i18next';
+import { useConfig } from '@openmrs/esm-framework';
 
 function HTSSummaryTiles({ launchWorkSpace }) {
   const { t } = useTranslation();
   const [todayPatientCount, setTodayPatientCount] = useState(0);
   const [positiveInLast14Days, setPositiveInLast14Days] = useState(0);
   const [linkedToCareInLast14Days, setLinkedToCareInLast14Days] = useState(0);
+  const config = useConfig();
```

Comment:
Am starting to think we can destructure this to reduce the long names.

Reply:
Done